cmake/Buildwhole-body-estimators.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia
+# Authors: Silvio Traversaro <silvio.traversaro@iit.it>
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(YARP QUIET)
+find_or_build_package(ICUB QUIET)
+find_or_build_package(iDynTree QUIET)
+
+set(whole-body-estimators_DEPENDS)
+
+list(APPEND whole-body-estimators_DEPENDS YARP)
+list(APPEND whole-body-estimators_DEPENDS ICUB)
+list(APPEND whole-body-estimators_DEPENDS iDynTree)
```

Comment:
Is there a way to handle the CMake option `IDYNTREE_USES_ICUB_MAIN` here? This is required for the compilation of `whole-body-estimators`. See https://github.com/robotology/whole-body-estimators/issues/46.

Reply:
Unfortunately there is no possibility of doing that. The iDynTree build by the superbuild is built with the `IDYNTREE_USES_ICUB_MAIN` , but if someone uses an external iDynTree with the IDYNTREE_USES_ICUB_MAIN set to OFF, there is no way to early fail at the moment.

Reply:
I see, thanks! . :+1: